suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -42,8 +42,13 @@ export const getAccountTransactions = (
     transactions: Record<string, WalletAccountTransaction[]>,
 ) => transactions[accountKey] || [];
 
-export const isPending = (tx: WalletAccountTransaction | AccountTransaction) =>
-    !!tx && (!tx.blockHeight || tx.blockHeight < 0);
+export const isPending = (tx: WalletAccountTransaction | AccountTransaction) => {
+    if (tx && tx.solanaSpecific?.status === 'confirmed') {
```

Comment:
nitpick (recommendation for the future): for consistency and better readability 
```suggestion
    if (tx?.solanaSpecific?.status === 'confirmed') {
```